hardware/ariadne/bootloaders/ariadne/main.c

```diff
@@ -133,6 +133,7 @@ int main(void)
 			}
 		}
 		/* Blink the notification led */
+		wdt_reset(); //Required so it doesn`t hang.
```

Comment:
Actually no. The one in L132 is required during flashing. The latter is
required on regular, non upload, boot.

Also sometimes it happens situations that the BL believe its flashing but
its actually not (noise...network broadcasts etc ?).

In my opnion both are needed.
On Jun 1, 2016 8:55 PM, "per1234" <notifications@github.com> wrote:

> In hardware/ariadne/bootloaders/ariadne/main.c
> <https://github.com/codebendercc/Ariadne-Bootloader/pull/31#discussion_r65463885>
> :
>
> > @@ -133,6 +133,7 @@ int main(void)
> >  			}
> >  		}
> >  		/* Blink the notification led */
> > +		wdt_reset(); //Required so it doesn`t hang.
>
> Doesn't this make the wdt_reset() call at
> https://github.com/codebendercc/Ariadne-Bootloader/blob/master/hardware/ariadne/bootloaders/ariadne/main.c#L132
> redundant?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/codebendercc/Ariadne-Bootloader/pull/31/files/fa21fce9a9a9abcb610efd7ef4925e4acd7dd5ac#r65463885>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFwaCKMIU2a7FCoZWmAr_qjyJyEil73Bks5qHhwMgaJpZM4IsED->
> .
>